### PR TITLE
integrating test sts aswi creds expire while object copy is in progress

### PR DESCRIPTION
**Pull request by hmaheswa**
this PR is to automate testing of sts aswi creds expire while object copy is in progress
hotfix bz: https://bugzilla.redhat.com/show_bug.cgi?id=2214981
polarion id: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83581390

ceph-qe-scripts PR: https://github.com/red-hat-storage/ceph-qe-scripts/pull/544

pass logs:
http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/PR_sts_hotfix_bz/5_3_test_sts_aswi_creds_expire_large_object_upload.console.log
http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/PR_sts_hotfix_bz/7_1_test_sts_aswi_creds_expire_large_object_upload.console.log
http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/PR_sts_hotfix_bz/5_3_test_sts_aswi.console.log

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by ckulal — COMMENTED** [review]
Looks good

File: suites/pacific/rgw/tier-2_rgw_sts_aswi.yaml [hunk 1]
```diff
@@ -195,3 +195,25 @@ tests:
         script-name: test_sts_aswi.py
         config-file-name: test_sts_aswi_s3_resource_tag_role_policy.yaml
         timeout: 500
+
```
Comment (anrao19) [inline suites/pacific/rgw/tier-2_rgw_sts_aswi.yaml:198]: remove extra line

File: suites/reef/rgw/tier-2_rgw_sts_aswi.yaml [hunk 2]
```diff
@@ -195,3 +195,25 @@ tests:
         script-name: test_sts_aswi.py
         config-file-name: test_sts_aswi_s3_resource_tag_role_policy.yaml
         timeout: 500
+
```
Comment (anrao19) [inline suites/reef/rgw/tier-2_rgw_sts_aswi.yaml:198]: same here

File: suites/pacific/rgw/tier-2_rgw_sts_aswi.yaml (hunk 1 above)
Comment (hmaheswa) [reply suites/pacific/rgw/tier-2_rgw_sts_aswi.yaml:198]: done @anrao19

File: suites/reef/rgw/tier-2_rgw_sts_aswi.yaml (hunk 2 above)
Comment (hmaheswa) [reply suites/reef/rgw/tier-2_rgw_sts_aswi.yaml:198]: done @anrao19

**Review by anrao19 — APPROVED** [review]
LGTM

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**